Resources/Private/JavaScript/Host/Containers/Drawer/index.js

```diff
@@ -49,51 +110,46 @@ export default class Drawer extends Component {
     }
 
     renderMenu() {
-        const staticMenuData = [{
-            icon: 'file',
-            title: moduleLabel('content_menu_menuPanel_content'),
-            children: [{
-                icon: 'globe',
-                title: 'test'
-            }]
-        }, {
-            icon: 'briefcase',
-            title: moduleLabel('management_label', 'Modules'),
-            children: [{
-                icon: 'th-large',
-                title: 'Workspaces'
-            }, {
-                icon: 'camera',
-                title: 'Media'
-            }, {
-                icon: 'calendar',
-                title: 'History'
-            }]
-        }];
-
-        return staticMenuData.map((item, index) => this.renderMenuItem(item, index));
+        const {menuData} = this.props;
+
+        return Object.keys(menuData).map(k => menuData[k]).map((item, index) => this.renderMenuItem(item, index));
```

Comment:
You're right of course. But I think we should even go one step further and use `Object.values`+polyfill or some other abstraction to avoid this pattern alltogether, since it is pretty counter-intuitive in and for itself.